model_compression_toolkit/target_platform_capabilities/target_platform/targetplatform2framework/attach2fw.py

```diff
@@ -35,22 +38,30 @@ def attach(self, tpc_model: TargetPlatformModel,
         """
 
         tpc = TargetPlatformCapabilities(tpc_model)
+        custom_opset2layer = custom_opset2layer if custom_opset2layer is not None else {}
```

Comment:
what is the reason not to define it {} in the function input instead of 'None'?

Reply:
This is bad practice to set mutable values as default arguments